packages/node_modules/@ciscospark/redux-module-spaces/src/actions.js

```diff
@@ -110,12 +114,15 @@ export function getSpaces(sparkInstance, options = {}) {
       items.forEach((space) => {
         if (!space.tags.includes(TAG_HIDDEN)) {
           spaces[space.id] = constructSpace(space, listOptions.deferDecrypt);
+          dispatch(storeUsers(space.participants.items));
```

Comment:
Should this go in an `else` after `if (space.decrypt)`?

Reply:
This is should happen even if we don't enable deferred decrypt. When we do defer, this line adds the encrypted users as placeholders. If we don't defer, this line will add the already decrypted users.

Reply:
Do we get a proper ID or email for encrypted users that will be updated after the decrypt?